ApplicationLibCode/ProjectDataModel/RimEclipseResultDefinition.cpp

```diff
@@ -150,10 +150,10 @@ RimEclipseResultDefinition::RimEclipseResultDefinition( caf::PdmUiItemInfo::Labe
     CAF_PDM_InitScriptableFieldNoDefault( &m_phaseSelection, "PhaseSelection", "Phases", "", "", "" );
     m_phaseSelection.uiCapability()->setUiLabelPosition( m_labelPosition );
 
-    CAF_PDM_InitScriptableField( &m_showOnlyVisibleTracersInLegend,
-                                 "ShowOnlyVisibleTracersInLegend",
+    CAF_PDM_InitScriptableField( &m_showOnlyVisibleCategoriesInLegend,
+                                 "ShowOnlyVisibleCategoriesInLegend",
```

Comment:
Possible back-compatibility issue? PdmField changes id. Maybe not that important though?

Reply:
Not in public release yet